instagram-page-react/src/pages/profilePage/organisms/posts/Posts.js

```diff
@@ -0,0 +1,34 @@
+import React, {useEffect, useState} from 'react';
+
+// Lodash
+import _map from 'lodash/map';
+
+//Styles
+import styles from './posts.module.css';
+
+// Containers
+import Post from "./molecules/post";
+
+const renderPost = (postNumber = 0) => {
+    return <Post
+        key={postNumber}
+        postNumber={postNumber}
+    />;
+}
+
+function Posts() {
+    const [postsNumber, setPostsNumber] = useState([]);
```

Comment:
This state will be an array of posts and not post numbers
`[
  { postPath: '/assets/.../.png' },
  { postPath: '/assets/.../.png' },
]`